src/components/StackQueueVisualizer.jsx

```
@@ -0,0 +1,875 @@
+          if (elements.length === 0) {
+            setCurrentLine(20);
+            setCurrentStep("Stack is empty");
+          } else {
+            setCurrentLine(22);
+            setCurrentStep("Stack has elements, proceeding with pop");
+          }
+          break;
+        case 4: // Pop operation
+          setCurrentLine(22);
+          setCurrentStep("Removing top element");
+          const poppedValue = elements[elements.length - 1];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)